UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Response/InternalServerError.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ExchangeSetService.Common.Models.Response
+{
+    [ExcludeFromCodeCoverage]
+    public class InternalServerError
+    {
+        public string CorrelationId { get; set; }
+        public string Details { get; set; }
```

Comment:
Kindly change this to public string Detail { get; set; } to match according to API spec.